src/_modules/common_values/outputs_general.tf

```diff
@@ -0,0 +1,27 @@
+output "resource_groups" {
```

Comment:
I don't get the logic behind this. Why does ITN map all the RGs that are not actually deployed in it? I though it was for consistency, but `acr`, `assets_cdn`, `dashboards` and `github_id` are not mapped

Reply:
It’s just that those were in the state, I’ve referenced the `common` configuration where they’re passed in the `local`. If those four aren’t needed, I can remove them, It’s just a name dictionary, so no issues will arise